SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/Http/NetworkStateIngestion.cs

```diff
@@ -1,124 +1,102 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.AppCenter.Ingestion.Models;
 
 namespace Microsoft.AppCenter.Ingestion.Http
 {
-    public class NetworkStateIngestion : IngestionDecorator
+    internal sealed class NetworkStateIngestion : IngestionDecorator
```

Comment:
why sealed?

Reply:
This is a decorator and other classes from should not be inherited from it. This is just guard.